src/coreclr/jit/gentree.h

```diff
@@ -5542,8 +5544,12 @@ struct GenTreeCall final : public GenTree
     union {
         // only used for CALLI unmanaged calls (CT_INDIRECT)
         GenTree* gtCallCookie;
+
         // gtInlineCandidateInfo is only used when inlining methods
-        InlineCandidateInfo*                 gtInlineCandidateInfo;
+        InlineCandidateInfo* gtInlineCandidateInfo;
+        // gtInlineCandidateInfoList is used when we have more than one GDV candidate
+        jitstd::vector<InlineCandidateInfo>* gtInlineCandidateInfoList;
```

Comment:
Decided to go with a vector instead of a linked list because we often access its elements by index, although, the maximum number of element is 5 and in 80% of cases calls are monomorphic so it doesn't really matter

Reply:
Can this be a vector of `InlineCandidateInfo*`? Looks like these are already heap allocated so we can avoid some struct copying.